__tests__/Unit/Components/Tasks/UserNotFound.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import UserNotFound from '@/components/tasks/SuggestionBox/UserNotFound';
+
+describe('Loader', () => {
+    it('renders the loading spinner', () => {
+        render(<UserNotFound />);
+        const userNotFound = screen.getByTestId('user_not_found');
+        expect(userNotFound).toBeInTheDocument();
```

Comment:
Is this checking users not found? If so please change the description of the test. Misleading.